cache/class-vary-cache.php

```diff
@@ -331,10 +374,16 @@ public static function set_cookie_expiry( int $expiry ) {
 		self::$cookie_expiry = $expiry;
 	}
 
+	public static function send_headers() {
+		self::$did_send_headers = true;
+
+		self::send_vary_headers();
+	}
+
 	/**
 	 * Add the vary cache headers to indicate that the response should be cached
 	 */
-	public static function add_vary_headers() {
+	private static function send_vary_headers() {
```

Comment:
Is there any reason we make the public method just a wrapper for the private function?

Reply:
Will be adding the cookie handling here so it made sense to have a general wrapper method.